docs/dev/agent_build.md

```diff
@@ -52,7 +52,7 @@ Also note that the trace agent needs to be built and run separately. For more in
 We use `pkg-config` to make compilers and linkers aware of Python. The required .pc files are
 provided automatically when building python through omnibus.
 
-By default, the Agent combines multiple functionalities into a single binary to reduce
+As an option, the Agent can combines multiple functionalities into a single binary to reduce
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
```suggestion
As an option, the Agent can combine multiple functionalities into a single binary to reduce
```